config/settings.yml

```diff
@@ -1306,6 +1306,7 @@
         :ems_metrics_collector_worker_openstack_network: {}
         :ems_metrics_collector_worker_redhat: {}
         :ems_metrics_collector_worker_vmware: {}
+        :ems_metrics_openstack_default_service: "auto"
```

Comment:
"auto" (or any other string except "gnocchi" or "ceilometer") means use gnocchi with ceilometer fallback, what might have different behavior in Openstack than without fallback.